instat/dlgDotPlot.vb

```diff
@@ -62,42 +62,39 @@ Public Class dlgDotPlot
         ucrDotPlotSelector.SetParameter(New RParameter("data", 0))
         ucrDotPlotSelector.SetParameterIsrfunction()
 
-        ucrPnlBinAxis.SetParameter(New RParameter("binaxis"))
+        ucrPnlBinAxis.SetParameter(New RParameter("binaxis", 0))

```

Comment:
Added parameter position and made it 0. This is the only parameter in the `geom_dotplot`